.travis.yml

```diff
@@ -1,24 +1,37 @@
 sudo: false
 language: php
 php:
-  - nightly
+  - 7.0
   - 5.6
   - 5.5
-  - 5.4
-  - hhvm
+
+env:
+  - TRAVIS_NODE_VERSION="4.2.3" CXX="g++-4.8"
+  - TRAVIS_NODE_VERSION="5.3.0" CXX="g++-4.8"
 
 matrix:
   allow_failures:
-    - php: nightly
+    - php: 5.5
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
 
 cache:
   directories:
     - bower_components
     - node_modules
+    - vendor
 
 install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
I'm assuming it's needed. I'm not sure, though; I didn't test without it. I did see that Travis supports nvm natively, but I assume that's only for node_js containers. 

I poached this from some random nerd online: http://austinpray.com/ops/2015/09/20/change-travis-node-version.html